Apps/WebClient/src/appsettings.json

```diff
@@ -65,8 +65,17 @@
             "Dependent": true
         },
         "HoursForDeletion": 720,
-        "MinPatientAge" : 12,
-        "MaxDependentAge": 12
+        "MinPatientAge": 12,
+        "MaxDependentAge": 12,
+        "OfflineMode": {
+            "StartDateTime": "2020/06/01 12:00:00PM",
+            "EndDateTime": "2020/06/01 12:00:00PM",
```

Comment:
Iso date formats translate better: ```2021-01-15T11:47:02-08:00

Reply:
These dates can be any format that C# can convert natively... since these are in dev now only I'll leave them as we will probably remove them before we're done.